src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -163,32 +167,51 @@ export async function lambdaHandler(
     });
   }
 
-  const recoverSessionService = dependencies.getRecoverSessionService(
+  const sessionService = dependencies.getSessionService(
     config.SESSION_TABLE_NAME,
     config.SESSION_TABLE_SUBJECT_IDENTIFIER_INDEX_NAME,
   );
 
   const recoverSessionServiceResponse =
-    await recoverSessionService.getAuthSessionBySub(
+    await sessionService.getAuthSessionBySub(
       parsedRequestBody.sub,
       parsedRequestBody.state,
       config.SESSION_RECOVERY_TIMEOUT,
     );
   if (recoverSessionServiceResponse.isError) {
-    return serverError500Responses;
+    return serverError500Response;
   }
 
   if (recoverSessionServiceResponse.value) {
     return sessionRecoveredResponse(parsedRequestBody.sub);
   }
 
-  return {
-    headers: { "Content-Type": "application/json" },
-    statusCode: 200,
-    body: JSON.stringify({
-      message: "Hello World",
-    }),
+  const { sub, client_id, govuk_signin_journey_id, redirect_uri, state } =
+    parsedRequestBody;
+  const { iss, aud } = jwtPayload;
+
+  const sessionId = randomUUID();
+
+  const sessionConfig = {
+    sessionId,
+    state,
+    sub,
+    clientId: client_id,
+    govukSigninJourneyId: govuk_signin_journey_id,
+    redirectUri: redirect_uri,
+    aud,
```

Comment:
This claim shouldn't be in the session table

Reply:
Gone